Makefile

```diff
@@ -52,17 +53,25 @@ clean:
 	rm -f $(BINARY_NAME)
 
 .PHONY: deps
-deps:
-	$(GOGET) ${u} -d
+deps: ensure-gopath
+	$(GOGET) github.com/swaggo/cli
+	$(GOGET) github.com/ghodss/yaml
+	$(GOGET) github.com/KyleBanks/depth
+	$(GOGET) github.com/go-openapi/jsonreference
+	$(GOGET) github.com/go-openapi/spec
 	$(GOGET) github.com/stretchr/testify/assert
 	$(GOGET) github.com/alecthomas/template
 
 .PHONY: devel-deps
-devel-deps:
+devel-deps: ensure-gopath
 	GO111MODULE=off $(GOGET) -v -u \
-		golang.org/x/lint/golint \
-		github.com/swaggo/swag/cmd/swag	\
-		github.com/swaggo/swag/gen
+		golang.org/x/lint/golint 
+
+.PHONY: ensure-gopath
+ensure-gopath:
+	mkdir -p ${GOPATH}/github.com/swaggo
+	if [ -L ${GOPATH}/github.com/swaggo/swag ]; then rm ${GOPATH}/github.com/swaggo/swag; fi
+	ln -s "$(shell pwd)"  ${GOPATH}/github.com/swaggo/swag
```

Comment:
I wonder why there needs to add `ensure-gopath` to check?

Reply:
ensure-gopath is only for go1.10

Reply:
If you don't like it there please let me know where I should put it.

Reply:
no worry. Just a question. 👍